examples/api/lib/widgets/heroes/hero.0.dart

```diff
@@ -41,17 +38,18 @@ class MyStatelessWidget extends StatelessWidget {
         ListTile(
           leading: Hero(
             tag: 'hero-rectangle',
-            child: _blueRectangle(const Size(50, 50)),
+            child: _box(const Size(50, 50)),
           ),
           onTap: () => _gotoDetailsPage(context),
-          title:
-              const Text('Tap on the icon to view hero animation transition.'),
+          title: const Text(
+            'Tap on the icon to view hero animation transition.',
+          ),
         ),
       ],
     );
   }
 
-  Widget _blueRectangle(Size size) {
+  Widget _box(Size size) {
```

Comment:
I think it would be a good example for the audience if this were a separate stateless widget. It's more efficient that way, and helps with segmentation of the code.

Reply:
Fore more context about widget methods vs classes: https://stackoverflow.com/a/53234826/8358501

Reply:
Thank you so much for your feedback, I will address this later today.

Reply:
I did this here earlier today

https://github.com/flutter/flutter/pull/103041/files#diff-044e0b5a642929f285ed1217ca886b8a710deaa1958deeae14dba77c629ddc0dR109